CMakeLists.txt

```diff
@@ -112,6 +107,23 @@ else ()
   add_definitions (-DHAVE_C99_MATH=0)
 endif ()
 
+find_package(Threads)
```

Comment:
find_package(Threads) is also at line 82. Probably the one at line 82 should be removed then ?